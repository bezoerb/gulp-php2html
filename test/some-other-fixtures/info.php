<?php
/*
 * grunt-php2html
 * https://github.com/bzoerb/grunt-php2html
 *
 * Copyright (c) 2013 Ben ZÃ¶rb
 * Licensed under the MIT license.
 */
?>
<html>
<head><title>TEST</title></head>
<body>
<h1>Simple Test without includes</h1>
<?php echo '<p>TEST</p>'; ?>
</body>
</html>
