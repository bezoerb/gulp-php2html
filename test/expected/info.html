<html>
<head><title>TEST</title></head>
<body>
<h1>Simple Test without includes</h1>
<p>TEST</p>
</body>
</html>